guidelines/releases-guidelines/02-release-process.md

```diff
@@ -3,39 +3,65 @@ title: Release process
 label: internal
 ---
 
-This document describes how to create a Kyma release using Prow.
+This document explains how a new Kyma and also new Kyma CLI release can be introduced.
 
-## Preparation
+## Kyma
 
-This section only applies to new major and minor versions. If you release a patch, skip the preparation and go to the [**Steps**][/#release-process-release-process-steps] section.
+### Release Information
 
-To prepare a release, define a new and remove the old release.
+A Kyma release consists of the following things:
+* Docker images for Kyma components
+* Github release
+  * Release artifacts such as source code and configuration
+* Git tag
+* Release branch
 
-### Define release jobs
+### Changes to test-infra repository
 
-Define release jobs on the `master` branch in the `test-infra` repository. To ensure every job name is unique, prefix it with `pre-rel{versionNumber}`. Remember to provide the version number without any periods. For example, to find all jobs for the 0.9 release, look for job names with the `pre-rel09` prefix. To learn how to define a release job for a component, read the following [document][https://github.com/kyma-project/test-infra/blob/master/docs/prow/release-jobs.md].
+#### Steps for new major/minor version
+
+Define release jobs on the `master` branch in the `test-infra` repository. To ensure every job name is unique, prefix it with `pre-rel{versionNumber}`. Remember to provide the version number without any periods. For example, to find all jobs for the 1.4 release, look for job names with the `pre-rel14` prefix. To learn how to define a release job for a component, read the following [document](https://github.com/kyma-project/test-infra/blob/master/docs/prow/release-jobs.md).
 
 1. Navigate to the `test-infra` repository.
-2. Define release jobs in the `prow/jobs/test-infra` directory in the `watch-pods.yaml` file.
-3. Define release jobs in the `prow/jobs/kyma` directory in the following files:
+1. Define release jobs in the `prow/jobs/test-infra` directory in the `watch-pods.yaml` file.
+1. Define release jobs in the `prow/jobs/kyma` directory in the following files:
```

Comment:
Did you intentionally put `1.` for 2 lines? If we intend to execute these steps to be run in parallel, I think dashes should be the way to go. Or `1. 2. 3. ....`

Reply:
As discussed in person: when using a markdown viewer each `1.` is automatically incremented